ibm/service/power/resource_ibm_pi_dhcp.go

```diff
@@ -101,7 +101,8 @@ func ResourceIBMPIDhcp() *schema.Resource {
 			Attr_DhcpNetworkDeprecated: {
 				Type:        schema.TypeString,
 				Computed:    true,
-				Description: "The ID of the DHCP Server private network (deprecated - replaced by network_id)",
+				Description: "The ID of the DHCP Server private network",
+				Deprecated:  "This field is deprecated, use network_id instead",
```

Comment:
Since it is deprecated we might as well take it out, I think it has been deprecated for a while

Reply:
Removed deprecated fields.